pkgs/applications/misc/buku/default.nix

```diff
@@ -31,7 +32,7 @@ with pythonPackages; buildPythonApplication rec {
     homepage = https://github.com/jarun/Buku;
     license = licenses.gpl3;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ matthiasbeyer ];
+    maintainers = with maintainers; [ matthiasbeyer infinisil ];
```

Comment:
Hey! You don't need to put yourself into the maintainers array, if you don't want to! If you want to, yes, feel free, I appreciate that! But I want you to know that you do not have to! :smile: